### ISSUE TITLE
[node_exporter] Custom prometheus metric created in file now require help

### ISSUE THREAD

**Issue by limed** [nubisproject/nubis-base#737]
With v0.15.0 upgrade creating a custom metric in directory `/var/lib/node_exporter/metrics` of the same name and but separated by  different files causes the node exporter to not create the node exporter to throw errors

```
ERRO[0002] error gathering metrics: 12 error(s) occurred:
* gathered metric family nubis_cron_end has help "Metric read from /var/lib/node_exporter/metrics/nubis_cron_fluent-watchdog.prom" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_start has help "Metric read from /var/lib/node_exporter/metrics/nubis_cron_fluent-watchdog.prom" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_duration has help "Metric read from /var/lib/node_exporter/metrics/nubis_cron_fluent-watchdog.prom" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_status has help "Metric read from /var/lib/node_exporter/metrics/nubis_cron_fluent-watchdog.prom" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_end has help "Metric read from file" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_start has help "Metric read from file" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_duration has help "Metric read from file" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_status has help "Metric read from file" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_duration has help "Metric read from file" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_status has help "Metric read from file" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_end has help "Metric read from file" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
* gathered metric family nubis_cron_start has help "Metric read from file" but should have "Metric read from /var/lib/node_exporter/metrics/nubis_cron_confd-watchdog.prom"
```

The fix is to include a `#HELP` line on each of the metric with the same format, this error at the moment is localize to only the `nubis_cron_*` because its the only instance where we have multiple files of the same metric

**Comment by limed** [comment]
This needs to be fixed when #720 is resolved

**Comment by limed** [comment]
Fixed in #738
